src/main/java/com/rultor/agents/req/StartsRequest.java

```diff
@@ -173,6 +190,42 @@ public boolean apply(final String input) {
     }
 
     /**
+     * Obtain proxy settings and create a Decrypt instance.
+     * @return Decrypt instance.
+     */
+    private Decrypt createDecrypt() {
```

Comment:
see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#method-names